README.md

```diff
@@ -162,18 +162,137 @@ Während der Übertragung der Daten zu den Servern wird in der Statuszeile (unte
 Damit die Daten, die der Sensor nach sensor.community schickt, von dem Server auch angenommen werden, muss man sich dort anmelden. Das geschieht über die Seite https://meine.luftdaten.info.
 Zuerst über den *Registrieren*-Knopf einen Account anlegen. Dann damit über *Login* einloggen und *Neuen Sensor registrieren* anklicken.
 Dann das Formular ausfüllen:
- * Erste Zeile, Sensor ID:
+ * Erste Zeile, Sensor ID:\
  Hier die Nummer (nur die Zahlen) der SSID des Sensors eingeben (z.B. bei dem Sensor ESP-51564452 also dann nur 51564452 eingeben)
- * Zweite Zeile, Sensor Board:
+ * Zweite Zeile, Sensor Board:\
  Hier *esp32* auswählen (über die kleinen Pfeile rechts)
- * Basisinformation:
+ * Basisinformation:\
  Hier die Adresse eingeben (mit dem Land!). Der interne Name des Sensors kann beliebig vergeben werden, muss aber eingegeben werden. Bitte den Haken bei **Indoor-Sensor** setzen, so lange der Sensor wirklich innen ist.
- * Zusätzliche Informationen:
+ * Zusätzliche Informationen:\
```

Comment:
these `\` are somehow ugly. is there another way to solve this?

Reply:
The other method are the two spaces - and you don't like that either

Reply:
right. guess we just need more .rst docs and less markdown README. :-)